### PR TITLE
Header and Navigation + Board

### PR DESCRIPTION
**Pull request by PedroFelipe**
- [x] Header and Navigation
- [x] Board

## [After] Header and Navigation 
screen shot 2017-01-17 at 9 57 03 am

## [Before] Board
imagem colada em 2017-1-18 10-41

### [After] Board
screen shot 2017-01-19 at 11 06 02 am

screen shot 2017-01-23 at 9 47 06 am

### Demo
https://cm-fulcrum-pr-95.herokuapp.com/projects/test-project

### CODE REVIEW

**Comment by talyssonoc** [comment]
I think the spacing here is missing a pixel on each side.

File: app/assets/stylesheets/_layout.scss [hunk 1]
```diff
@@ -1,4 +1,14 @@
...
+  background-color: $lightgrey-7;
+}
+
+.container-fluid {
+  padding-left: 0;
+  padding-right: 0;
+}
+
+.board {
+  background-color: #1f2021;
```
Comment (marclerodrigues) [inline app/assets/stylesheets/_layout.scss]: Variable here.

File: app/assets/stylesheets/_navbar.scss [hunk 2]
```diff
@@ -1,66 +1,107 @@
...
+    margin-left: 5px;
+
+    &:after {
+      content: '';
+      position: absolute;
+      top: 1px;
+      left: 0;
+      width: 1px;
+      height: 20px;
+      background-color: rgba($darkgrey-6, .25);
```
Comment (marclerodrigues) [inline app/assets/stylesheets/_navbar.scss]: Some times you use with zero `0.x` some times just the dot `.x` for representing numbers. Would be better to choose a standard.

File: app/assets/stylesheets/_icons.scss [hunk 3]
```diff
@@ -12,23 +12,26 @@ i {
   vertical-align: middle;
 }
 
-.icon{
-  width:16px;
-  height:16px;
-  padding:0;
-  display:inline-block;
-  text-indent:-9999px;
+.icon {
```
Comment (talyssonoc) [inline app/assets/stylesheets/_icons.scss]: 👏

File: app/assets/stylesheets/_navbar.scss [hunk 4]
```diff
@@ -1,66 +1,107 @@
...
+        content: '';
+        position: absolute;
+        top: 9px;
+        right: 5px;
+        display: inline-block;
+        width: 0;
+        height: 0;
+        vertical-align: middle;
+        border-top: 4px dashed;
+        border-top: 4px solid \9;
```
Comment (marclerodrigues) [inline app/assets/stylesheets/_navbar.scss]: Is this correct these two lines defining border-top?

File: app/assets/stylesheets/_screen.scss [hunk 5]
```diff
@@ -113,7 +101,7 @@ table.stories {
 
 table.stories td {
   vertical-align: top;
-  background-color: #ccc;
+  background-color: #484f57;
```
Comment (marclerodrigues) [inline app/assets/stylesheets/_screen.scss]: Variable here.

File: app/assets/stylesheets/_screen.scss [hunk 6]
```diff
@@ -124,49 +112,55 @@ div.storycolumn {
...
-  color: #eee;
-  padding: 0.4em 0.2em 0.3em 0.7em;
-  text-transform: uppercase;
   @include border-radius(2px 2px 0 0);
-  a.toggle-title{
-    color: white;
+  padding: 8px 10px;
+  font-size: 12px;
+  color: $white;
+  background: #2C2D2F;
```
Comment (marclerodrigues) [inline app/assets/stylesheets/_screen.scss]: Variable here.

File: app/assets/stylesheets/_screen.scss [hunk 7]
```diff
@@ -124,49 +112,55 @@ div.storycolumn {
...
 }
 
 /* Story types */
 div.story {
-  padding: 0.5em 0.5em 0.9em;
+  padding: 7px 8px;
   background-repeat: no-repeat;
   background-position: 4px 4px;
-  background-color: #eee;
+  background-color: #f4f4f4;
```
Comment (marclerodrigues) [inline app/assets/stylesheets/_screen.scss]: Variable here.

File: app/assets/stylesheets/_screen.scss [hunk 8]
```diff
@@ -124,49 +112,55 @@ div.storycolumn {
...
+  border-top: 1px solid $white;
 
   &:not(.editing):not(.unestimated) {
     min-height: 37px;
   }
 }
+
 div.story:hover {
-  background-color: #ddd;
+  background-color: #e6e6e6;
```
Comment (marclerodrigues) [inline app/assets/stylesheets/_screen.scss]: Variable here.

File: app/assets/stylesheets/_screen.scss [hunk 9]
```diff
@@ -266,12 +254,30 @@ div.story-title abbr.initials {
   }
 }
 
-.story.release {
-  background-color: $darkgrey-2;
-  color: white;
+.story.started:not(.release) {
+  background-color: #f3f3d1;
```
Comment (marclerodrigues) [inline app/assets/stylesheets/_screen.scss]: Variable here.

File: app/assets/stylesheets/_screen.scss [hunk 10]
```diff
@@ -266,12 +254,30 @@ div.story-title abbr.initials {
...
+  border-bottom: 1px solid #dbdbb0;
```
Comment (marclerodrigues) [inline app/assets/stylesheets/_screen.scss]: The same here.

File: app/assets/stylesheets/_screen.scss [hunk 11]
```diff
@@ -266,12 +254,30 @@ div.story-title abbr.initials {
...
 }
-.story.release:hover {
-  background-color: #222;
+
+.story.release {
+  color: $white;
+  background-color: #407aa5;
```
Comment (marclerodrigues) [inline app/assets/stylesheets/_screen.scss]: And here.

File: app/assets/stylesheets/_screen.scss [hunk 12]
```diff
@@ -266,12 +254,30 @@ div.story-title abbr.initials {
...
+  border-top-color: #5982a7;
```
Comment (marclerodrigues) [inline app/assets/stylesheets/_screen.scss]: And here.

File: app/assets/stylesheets/_screen.scss [hunk 13]
```diff
@@ -266,12 +254,30 @@ div.story-title abbr.initials {
...
+  border-bottom-color: #306494;
```
Comment (marclerodrigues) [inline app/assets/stylesheets/_screen.scss]: And here.

File: app/assets/stylesheets/_screen.scss [hunk 14]
```diff
@@ -266,12 +254,30 @@ div.story-title abbr.initials {
...
+
+  &:hover {
+    background-color: #306494;
```
Comment (marclerodrigues) [inline app/assets/stylesheets/_screen.scss]: And here.

File: app/assets/stylesheets/_navbar.scss (hunk 4 above)
Comment (PedroFelipe) [inline app/assets/stylesheets/_navbar.scss]: It is! Check it out:
screen shot 2017-01-19 at 2 07 14 pm

File: app/assets/stylesheets/_navbar.scss [hunk 15]
```diff
@@ -59,6 +59,40 @@
...
+      margin-top: 10px;
+      width: 300px;
+      font-size: 11px;
+      background-color: lighten($aluminium-6, 3);
+
+      .form-group {
+        margin: 0;
+      }
+
+      p {
```
Comment (talyssonoc) [inline app/assets/stylesheets/_navbar.scss]: Can we use a class for that?

File: app/assets/javascripts/views/project_velocity_override_view.js [hunk 16]
```diff
@@ -46,6 +46,10 @@ module.exports = Backbone.View.extend({
       that.clickOverlayOff();
     });
     $('.click-overlay').show();
+
+    setTimeout(function() {
+      $('.click-overlay').css('top', $('nav.navbar').height());
```
Comment (talyssonoc) [inline app/assets/javascripts/views/project_velocity_override_view.js]: 😢  isn't it possible to just use some relative position instead?

Comment (PedroFelipe) [inline app/assets/javascripts/views/project_velocity_override_view.js]: removed

File: app/assets/javascripts/global_listeners.js [hunk 17]
```diff
@@ -103,8 +103,11 @@ $(function() {
   $('.tag-tooltip').tooltip();
 
   $('.locale-change').on('change', function(e) {
+    console.log('locale changed');
```
Comment (talyssonoc) [inline app/assets/javascripts/global_listeners.js]: I think it's not supposed to be here 😛

File: app/assets/stylesheets/_navbar.scss [hunk 18]
```diff
@@ -1,66 +1,182 @@
...
-        .nav-link {
-          background-color: transparent;
-          border-bottom: 2px solid $darkred;
-          color: $darkred;
-        }
+  &-logo {
+    padding: 10px 15px;
+  }
+
+  &-nav > li > a {
```
Comment (talyssonoc) [inline app/assets/stylesheets/_navbar.scss]: Is there some way to avoid that?

File: app/assets/stylesheets/_navbar.scss [hunk 19]
```diff
@@ -1,66 +1,182 @@
...
+  &-search {
+    margin: 10px;
+  }
+
+  .velocity {
+    position: relative;
+    padding-left: 15px;
+    margin-left: 5px;
+
+    & > span:first-child {
```
Comment (talyssonoc) [inline app/assets/stylesheets/_navbar.scss]: Can we avoid that?

File: app/assets/javascripts/global_listeners.js (hunk 17 above)
Comment (PedroFelipe) [inline app/assets/javascripts/global_listeners.js]: oops

File: app/assets/stylesheets/_navbar.scss (hunk 18 above)
Comment (PedroFelipe) [inline app/assets/stylesheets/_navbar.scss]: if I write on another way, I wouldn't be able to override Bootstrap's own selector :/ it has priority being written like this

**Pull request merged**